whois-api/src/main/java/net/ripe/db/whois/api/nrtm4/NrtmExceptionMapper.java

```diff
@@ -26,6 +27,10 @@ public Response toResponse(final Exception exception) {
             return createErrorResponse(HttpServletResponse.SC_NOT_FOUND, exception.getMessage());
         }
 
```

Comment:
Is it also being logged as error in sentry ?

Reply:
I don't see any alert in sentry and I've just run the curl there